las_geoh5/export_las.py

```diff
@@ -0,0 +1,209 @@
+#  Copyright (c) 2023 Mira Geoscience Ltd.
+#
+#  This file is part of my_app project.
+#
+#  All rights reserved.
+#
+
+from __future__ import annotations
+
+from pathlib import Path
+
+from geoh5py.data import ReferencedData
+from geoh5py.groups import PropertyGroup
+from geoh5py.objects import Drillhole
+from geoh5py.shared.concatenation import ConcatenatedDrillhole
+from lasio import HeaderItem, LASFile
+
+
+def add_well_data(file: LASFile, drillhole: Drillhole | ConcatenatedDrillhole):

```

Comment:
**suggestion (typing):** add type hint for returned type
```suggestion
def add_well_data(file: LASFile, drillhole: Drillhole | ConcatenatedDrillhole) -> LASFile:

```

Reply:
Done